libs/core/src/integration/events.schemas.ts

```diff
@@ -240,3 +240,11 @@ export const FarcasterReplyCastCreated = FarcasterCast.describe(
 export const FarcasterVoteCreated = FarcasterAction.extend({
   contest_address: z.string(),
 }).describe('When a farcaster action is initiated on a cast reply');
+
+export const TokenLaunched = EventMetadata.extend({
```

Comment:
How is this event used if you don't have a mapper for it and there is no custom code for it in evm chain-events?

Is the `kind` required here?

Reply:
migrated to evm chain event pattern